run-integration-tests.py

```diff
@@ -253,6 +253,11 @@ def run_pip_installation_tests(root_dir, version, use_testpypi, extra_maven_repo
         install_cmd = ["pip", "install",
                        "--extra-index-url", "https://test.pypi.org/simple/",
                        delta_pip_name_with_version]
+    elif use_localpypi:
+        pip_wheel_file_name = "%s-%s-py3-none-any.whl" % \
+                              (delta_pip_name.replace("-", "_"), str(version))
+        pip_wheel_file_path = os.path.join(use_localpypi, pip_wheel_file_name)
+        install_cmd = ["pip", "install", pip_wheel_file_path]
```

Comment:
would you need to uninstall the previous one? what if you run this multiple times ... but the local pypi has changed in between? would it juse use the old one?

Reply:
It is uninstalled before we even come here: see L#248